core/src/jsMain/kotlin/dev/fritz2/dom/html/elements.kt

```diff
@@ -1315,19 +1314,87 @@ class Path(id: String? = null, baseClass: String? = null, job: Job, scope: Scope
+        val target = into?.apply(SET_MOUNT_POINT_DATA_ATTRIBUTE)
+            ?: div(MOUNT_POINT_STYLE_CLASS, content = SET_MOUNT_POINT_DATA_ATTRIBUTE)
+
+        val mountContext = MountContext(Job(job), target)
+
+        mountSimple(job, this) {
+            mountContext.job.cancelChildren()
+            mountContext.runBeforeUnmounts()
+            target.domNode.clear()
+            content(mountContext, it)
+            mountContext.runAfterMounts()
+        }
+    }
+
+    /**
+     * Renders each element of a [Flow]s content.
+     * Internally the [Patch]es are determined using Myer's diff-algorithm.
+     * This allows the detection of moves. Keep in mind, that no [Patch] is derived,
+     * when an element stays the same, but changes its internal values.
+     *
+     * @param idProvider function to identify a unique entity in the list
+     * @param into target to mount content to. If not set a child div is added to the [Tag] this method is called on
+     * @param content [RenderContext] for rendering the data to the DOM
+     */
+    fun <V> Flow<List<V>>.renderEach(
+        idProvider: IdProvider<V, *>? = null,
+        into: Tag<HTMLElement>? = null,
+        content: RenderContext.(V) -> Tag<HTMLElement>
+    ) {
+        mountPatches(into, this) { upstreamValues, mountPoints ->
+            upstreamValues.scan(Pair(emptyList(), emptyList())) { acc: Pair<List<V>, List<V>>, new ->
+                Pair(acc.second, new)
+            }.map { (old, new) ->
+                val diff = if (idProvider != null) Myer.diff(old, new, idProvider) else Myer.diff(old, new)
+                diff.map { patch ->
+                    patch.map(job) { value, newJob ->
+                        val mountPoint = BuildContext(newJob, scope)
+                        content(mountPoint, value).also {
+                            mountPoints[it.domNode] = mountPoint
+                        }
+                    }
+                }
+            }
+        }
+    }
+
+    /**
+     * Renders each element of a [Store]s [List] content.
+     * Internally the [Patch]es are determined using Myer's diff-algorithm.
+     * This allows the detection of moves. Keep in mind, that no [Patch] is derived,
+     * when an element stays the same, but changes its internal values.
+     *
+     * @param idProvider function to identify a unique entity in the list
+     * @param into target to mount content to. If not set a child div is added to the [Tag] this method is called on
+     * @param content [RenderContext] for rendering the data to the DOM
+     */
+    fun <V> Store<List<V>>.renderEach(
+        idProvider: IdProvider<V, *>,
+        into: Tag<HTMLElement>? = null,
+        content: RenderContext.(Store<V>) -> Tag<HTMLElement>
+    ) {
+        data.renderEach(idProvider, into) { value ->
+            content(this@renderEach.sub(value, idProvider))
+        }
+    }
```

Comment:
Is this function really needed? It's only a convenience function that may confuse!?